astroquery/ukidss/core.py

```diff
@@ -64,26 +33,30 @@ class UkidssClass(QueryWithLogin):
                    'difference': 'diff', 'leavstack': 'leavstack',
                    'all': 'all'}
 
-    ukidss_programmes_short = {'LAS': 101,
-                               'GPS': 102,
-                               'GCS': 103,
-                               'DXS': 104,
-                               'UDS': 105, }
+    programmes_short = {'LAS': 101,
+                        'GPS': 102,
+                        'GCS': 103,
+                        'DXS': 104,
+                        'UDS': 105, }
 
-    ukidss_programmes_long = {'Large Area Survey': 101,
-                              'Galactic Plane Survey': 102,
-                              'Galactic Clusters Survey': 103,
-                              'Deep Extragalactic Survey': 104,
-                              'Ultra Deep Survey': 105}
+    programmes_long = {'Large Area Survey': 101,
+                       'Galactic Plane Survey': 102,
+                       'Galactic Clusters Survey': 103,
+                       'Deep Extragalactic Survey': 104,
+                       'Ultra Deep Survey': 105}
 
-    all_databases = ("UKIDSSDR9PLUS", "UKIDSSDR8PLUS", "UKIDSSDR7PLUS",
+    all_databases = ("UKIDSSDR10PLUS", "UKIDSSDR9PLUS",
+                     "UKIDSSDR8PLUS", "UKIDSSDR7PLUS",
                      "UKIDSSDR6PLUS", "UKIDSSDR5PLUS", "UKIDSSDR4PLUS",
                      "UKIDSSDR3PLUS", "UKIDSSDR2PLUS", "UKIDSSDR1PLUS",
                      "UKIDSSDR1", "UKIDSSEDRPLUS", "UKIDSSEDR", "UKIDSSSV",
                      "WFCAMCAL08B", "U09B8v20120403", "U09B8v20100414")
 
+    # needed for some WFAU queries, not for UKIDSS
+    archive = None
+
     def __init__(self, username=None, password=None, community=None,
-                 database='UKIDSSDR7PLUS', programme_id='all'):
+                 database='UKIDSSDR10PLUS', programme_id='all'):
         super(UkidssClass, self).__init__()
         self.database = database
```

Comment:
why duplicate these here from the baseclass rather than feed in to the ``super()`` call above?